common/util/cmd_positional_arguments.cc

```diff
@@ -0,0 +1,128 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "common/util/cmd_positional_arguments.h"
+
+#include <string>
+#include <vector>
+
+#include "absl/status/status.h"
+#include "absl/strings/str_split.h"
+#include "absl/strings/string_view.h"
+
+namespace verible {
+
+// Sets a file argument.
+absl::Status CmdPositionalArguments::SetFile(absl::string_view file) {
+  files_.push_back(file);
+  return absl::OkStatus();
+}
+
+// Sets a define arguments.
+absl::Status CmdPositionalArguments::SetDefine(
+    std::pair<absl::string_view, absl::string_view> define) {
+  defines_.push_back(define);
+  return absl::OkStatus();
+}
+
+// Sets a include directory argument.
+absl::Status CmdPositionalArguments::SetIncludeDir(
+    absl::string_view include_dir) {
+  include_dirs_.push_back(include_dir);
+  return absl::OkStatus();
+}
+
+// Gets include directories arguments.
+std::vector<absl::string_view> CmdPositionalArguments::GetIncludeDirs() {
+  return include_dirs_;
+}
+
+std::vector<std::pair<absl::string_view, absl::string_view>>
+
+// Gets macro defines arguments.
+CmdPositionalArguments::GetDefines() {
```

Comment:
The return value and the function header got separate here, which looks very odd. I guess this might've happened due to clang-formatter got confused. If you put the comment before the return value, things should be fine.

Having said this, this simple function is probably best implemented in the header.